mongo_orchestration/servers.py

```diff
@@ -81,8 +81,11 @@ def __init_compressors(self, config):
         """Conditionally enable compression in the Server's config file."""
         compressors = config.get('networkMessageCompressors')
         if compressors is None:
-            # SERVER-27310 added zlib support in 3.5.9.
-            if self.version >= (3, 5, 9):
+            if self.version >= (4, 1, 7):
+                # SERVER-38168 added zstd support in 4.1.7.
+                config['networkMessageCompressors'] = 'zstd,zlib,snappy,noop'
```

Comment:
Does the order here matter at all? Hmm, I suppose the first one will be the one that the cluster uses to communicate internally. Should `snappy` appear first in that case?

Reply:
Since we're only enabling it on 4.1+ I don't think it matters.